pkg/pillar/types/zedmanagertypes.go

```diff
@@ -152,16 +153,17 @@ func (config AppInstanceConfig) VerifyFilename(fileName string) bool {
 
 // Indexed by UUIDandVersion as above
 type AppInstanceStatus struct {
-	UUIDandVersion     UUIDandVersion
-	DisplayName        string
-	DomainName         string // Once booted
-	Activated          bool
-	ActivateInprogress bool     // Needed for cleanup after failure
-	FixedResources     VmConfig // CPU etc
-	StorageStatusList  []StorageStatus
-	EIDList            []EIDStatusDetails
-	OverlayNetworks    []OverlayNetworkStatus
-	UnderlayNetworks   []UnderlayNetworkStatus
+	UUIDandVersion      UUIDandVersion
+	DisplayName         string
+	DomainName          string // Once booted
+	Activated           bool
+	ActivateInprogress  bool     // Needed for cleanup after failure
+	FixedResources      VmConfig // CPU etc
+	StorageStatusList   []StorageStatus
+	VolumeRefStatusList []VolumeRefStatus
+	EIDList             []EIDStatusDetails
+	OverlayNetworks     []OverlayNetworkStatus
+	UnderlayNetworks    []UnderlayNetworkStatus
```

Comment:
Looks like identation only change because you removed a blank line

Reply:
Added ```VolumeRefStatusList``` in ```AppInstanceStatus ```